rich/console.py

```diff
@@ -294,7 +294,7 @@ def __enter__(self) -> "Capture":
         self._console.begin_capture()
         return self
 
-    def __exit__(self, exc_type, exc_val, exc_tb) -> None:
+    def __exit__(self, exc_type: Any, exc_val: Any, exc_tb: Any) -> None:
```

Comment:
I can never remember the types here, which is why omitted them but let's be accurate since we are --strict.

This is the types, copied from another project. Python2 syntax. 

``` python
    def __exit__(
        self,
        exc_type,  # type: Optional[Type[BaseException]]
        exc_value,  # type: Optional[BaseException]
        traceback,  # type: Optional[TracebackType]
    ):
```